utils/wxrc/wxrc.cpp

```diff
@@ -260,23 +260,34 @@ int XmlResApp::OnRun()
 {
     static const wxCmdLineEntryDesc cmdLineDesc[] =
     {
-        { wxCMD_LINE_SWITCH, "h", "help",  "show help message", wxCMD_LINE_VAL_NONE, wxCMD_LINE_OPTION_HELP },
-        { wxCMD_LINE_SWITCH, "v", "verbose", "be verbose" },
-        { wxCMD_LINE_SWITCH, "e", "extra-cpp-code",  "output C++ header file with XRC derived classes" },
-        { wxCMD_LINE_SWITCH, "c", "cpp-code",  "output C++ source rather than .rsc file" },
-        { wxCMD_LINE_SWITCH, "p", "python-code",  "output wxPython source rather than .rsc file" },
-        { wxCMD_LINE_SWITCH, "g", "gettext",  "output list of translatable strings (to stdout or file if -o used)" },
-        { wxCMD_LINE_OPTION, "n", "function",  "C++/Python function name (with -c or -p) [InitXmlResource]" },
-        { wxCMD_LINE_OPTION, "o", "output",  "output file [resource.xrs/cpp]" },
-        { wxCMD_LINE_SWITCH, "",  "validate", "check XRC correctness (in addition to other processing)" },
-        { wxCMD_LINE_SWITCH, "",  "validate-only", "check XRC correctness and do nothing else" },
-        { wxCMD_LINE_OPTION, "",  "xrc-schema", "RELAX NG schema file to validate against (optional)" },
+        { wxCMD_LINE_SWITCH, "h", "help", "show help message",
+              wxCMD_LINE_VAL_NONE, wxCMD_LINE_OPTION_HELP },
+        { wxCMD_LINE_SWITCH, "v", "verbose", "be verbose",
+              wxCMD_LINE_VAL_NONE, 0x0 },
```

Comment:
I really, really dislike this change (and a similar one in another commit). These extra `wxCMD_LINE_VAL_NONE, 0` harm code readability and are completely useless. I think it would be better to just disable the warning instead.

More generally, it'd be nice to have some newer API which allows to define the command line options without generating a warning but without having to specify all the fields neither...